signature-specification.md

```diff
@@ -135,7 +135,7 @@ Notary v2 requires the signature envelope to support the following signed attrib
 These attributes are considered unsigned with respect to the signing key that generates the signature. These attributes are typically signed by a third party (e.g. CA, TSA).
 
 - **Certificate Chain**: This is a REQUIRED attribute that contains the ordered list of X.509 public certificates associated with the signing key used to generate the signature. The ordered list starts with the signing certificates, any intermediate certificates and ends with the root certificate. The certificate chain MUST be authenticated against a trust store as part of signature validation. Specific requirements for the certificates in the chain are provided [here](#certificate-requirements).
-- **Timestamp signature** : An OPTIONAL counter signature which provides [trusted timestamp](#signing-time)e.g. Time Stamp Authority (TSA) generated timestamp signature. Only [RFC3161](ietf-rfc3161) compliant TimeStampToken are currently supported.
+- **Timestamp signature** : An OPTIONAL counter signature which provides [authentic timestamp](#signing-time)e.g. Time Stamp Authority (TSA) generated timestamp signature. Only [RFC3161](ietf-rfc3161) compliant TimeStampToken are currently supported.
 - **Signing Agent**: An OPTIONAL claim that provides the identifier of the software (e.g. Notation) that produced the signature on behalf of the user. It is an opaque string set by the software that produces the signature. It's intended primarily for diagnostic and troubleshooting purposes, this attribute is unsigned, the verifier MUST NOT validate formatting, or fail validation based on the content of this claim. The suggested format is one or more tokens of the form `{id}/{version}` containing identifier and version of the software, seperated by spaces. E.g. “notation/1.0.0”, “notation/1.0.0 com.example.nv2plugin/0.8”.
```

Comment:
Same as https://github.com/notaryproject/notaryproject/pull/164/files#r918406206
Why is it duplicated?
This description at least provides some details around troubleshooting. I really like troubleshooting data. But, I imagine many internal tools may not wish to publish their toolset. 

Just curious if we really need this.

Reply:
This is the Notary v2 signature spec, the other doc is the JWS "implementation" of it. This doc provides context. 

>But, I imagine many internal tools may not wish to publish their toolset.
By internal tools do you mean the plugin?

Reply:
Yeah, I'm referring to internal companies that may not want to publish their internal info. It is optional, so deferring.